scripts/aws/cfn/microservice.yml

```diff
@@ -1241,6 +1270,129 @@ Resources:
+  #       - nodejs18.x
+  #     Content:
+  #       S3Bucket: !Ref LambdasBucketName
+  #       S3Key: !Sub "${BucketBasePath}/raddBaselining_libs.zip"
+  #     Description: NodeJs Libs layer
+  #     LayerName: raddBaselining-libs-layer
+
+ # Lambda function role
+  PnRaddBaseliningLambdaStackRole:
+    Type: AWS::IAM::Role
+    Properties:
+      RoleName: !Sub ${ProjectName}-RaddBaseliningLambdaRole
+      AssumeRolePolicyDocument:
+        Version: 2012-10-17
+        Statement:
+          - Effect: Allow
+            Principal:
+              Service:
+                - lambda.amazonaws.com
+            Action:
+              - sts:AssumeRole
+      Path: /
+      Policies:
+        - PolicyName: "root"
+          PolicyDocument:
+            Version: 2012-10-17
+            Statement:
+              - Sid: AllowLambdaToCreateLogs
+                Action:
+                  - logs:CreateLogStream
+                  - logs:PutLogEvents
+                Effect: Allow
+                Resource: "*"
+              - Sid: AllowLambdaToCreateLogGroup
+                Action:
+                  - logs:CreateLogGroup
+                Effect: Allow
+                Resource: !Sub "arn:aws:logs:${AWS::Region}:${AWS::AccountId}:*"
+              - Sid: AllowLambdaReadDynamoDb
+                Action:
+                  - dynamodb:GetItem
+                Effect: Allow
+                Resource: 
+                  - !Ref AuthJwtAttributesTableArn
+              - Sid: AllowLambdaScanDynamoDb
+                Action:
+                  - dynamodb:Scan
+                Effect: Allow
+                Resource: 
+                  - !Ref AuthJwtIssuersTableArn
+      ManagedPolicyArns:
+        - arn:aws:iam::aws:policy/service-role/AWSLambdaVPCAccessExecutionRole
+
+  PnRaddBaseliningLambdaStack:
+    Type: AWS::CloudFormation::Stack
+    # UpdateReplacePolicy: Retain
+    # DeletionPolicy: Delete
+    Properties:
+      TemplateURL: !Sub "${TemplateBucketBaseUrl}/fragments/lambda.yaml"
+      Parameters:
+        FunctionName: !Ref PnRaddBaseliningLambdaName
+        Runtime: nodejs18.x
+        Handler: "index.handler"
+        MemorySize: 256
+        Timeout: 10
+        RoleArn: !GetAtt PnRaddBaseliningLambdaStackRole.Arn
+        EnvVariables:
+          !Sub [
+              "AUTH_JWT_ATTRIBUTE_TABLE=${auth_jwt_attribute_table}###AUTH_JWT_ISSUER_TABLE=${auth_jwt_issuer_table}###MAX_ATTRIBUTES_AGE_DAYS=${max_attribute_age_days}",
+              {
+                auth_jwt_attribute_table: !Ref AuthJwtAttributesTableName,
+                auth_jwt_issuer_table: !Ref AuthJwtIssuersTableName,
+                max_attribute_age_days: !Ref MaxAttributeAgeDays
+              },
+            ]
+       # Layer1: !Ref PnRaddBaseliningLambdaLibrariesLayer
+        FunctionBucketName: !Ref LambdasBucketName
+        FunctionBucketKey: !Sub "${BucketBasePath}/raddBaselining.zip"
+  
+ 
+
+  # Action Remover scheduler BEGIN
+  RaddBaseliningSchedulerRole:
```

Comment:
@abertu75 manca la lambda permission (AWS::Lambda::Permission)affinché possa essere invocata dal servizio scheduler di AWS? non sono sicuro, se l'avete provata in ninja deploy allora mi sbaglio.

Reply:
si, deployata in ninna e funziona

Reply:
@abertu75  però così può essere invocata potenzialmente da chiunque mentre con il lambda permission sei sicuro che solo lo scheduler possa eseguirla.